integration_tests/base_test.py

```diff
@@ -81,10 +82,16 @@ def _assert_run_fields(self, run, output_path):
         assert run["tool"] == "codemodder-python"
         assert run["version"] == __version__
         assert run["elapsed"] != ""
-        assert (
-            run["commandLine"]
-            == f"codemodder {SAMPLES_DIR} --output {output_path} --codemod-include={self.codemod_instance.name} --path-include={self.code_path}"
-        )
+        if self.sonar_issues_json:
+            assert (
```

Comment:
Just a small thing but feels like it might be a bit cleaner with something like:
```python
expected_cmd = base_cmd + ("--sonar-issues-json ..." if self.sonar_issues_json else "")
```

But I just have an irrational aversion to `if` blocks 😆